src/bvals/cc/bnd_info.cpp

```diff
@@ -250,8 +250,15 @@ int GetBufferSize(std::shared_ptr<MeshBlock> pmb, const NeighborBlock &nb,
 }
 
 BndInfo BndInfo::GetSendBndInfo(std::shared_ptr<MeshBlock> pmb, const NeighborBlock &nb,
-                                std::shared_ptr<CellVariable<Real>> v) {
+                                std::shared_ptr<CellVariable<Real>> v,
+                                CommBuffer<buf_pool_t<Real>::owner_t> *buf) {
```

Comment:
Was it a point of conversation during the Parthenon workshop to get rid of/avoid naked pointers from now on?

Reply:
I don't believe so. I think naked pointers are still pretty common---I think the style is do naked pointers when passing into a function that does not modify pointer scope?

Reply:
yes, I think the standard wisdom is to pass raw pointers to a function if the function does not cause something to take ownership of the resource for longer than the lifetime of the function itself.